data/governments.txt

```diff
@@ -546,6 +546,20 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered Challenger)"
+	"display name" "Unfettered Heshi"
+	swizzle 5
+	"crew attack" 1.4
+	"crew defense" 2.2
+
+	"player reputation" -1000
+	"bribe" .0001
```

Comment:
This seems smaller than other `bribe` values I've seen. (Probably doesn't matter)

Reply:
It's on purpose - mb it makes bribe and peace to attack again easier but you're also always meant to be able to call it off (and you have to win in the day so it's fine)
this battle here is hard enough that I intentionally let some cheeses open x)

Reply:
Does low bribe value mean more or less expensive?

Reply:
It's a thing so you can say, "Hey! I need a break!" ?

Reply:
yeah at least for the https://github.com/endless-sky/endless-sky/pull/6416 main campaign where it's explained how duels work (but that PR will get split, it's mostly there for playtesting the whole thing)
I could change it tho, I just dont mind ppl abusing it a bit, if they do too much I'll code smth so you can only bribe once per day for the duellists

Reply:
The bribe thing is fine, I was just wondering about it.

Reply:
Ik, I'm happy to clarify things